kura/distrib/config/kura.build.properties

```diff
@@ -1,128 +1,128 @@
+org.eclipse.kura.linux.position.version=1.2.0
+org.eclipse.kura.linux.usb.version=1.2.0
+org.eclipse.kura.linux.watchdog.version=1.2.0
+org.eclipse.kura.net.admin.version=1.2.0
+org.eclipse.kura.util.version=1.3.0
+org.eclipse.kura.protocol.can.version=3.2.0
+org.eclipse.kura.protocol.modbus.version=2.2.0
+org.eclipse.kura.demo.heater.version=1.2.0
+org.eclipse.kura.demo.modbus.version=2.2.0
+org.eclipse.kura.example.project.version=1.2.0
+org.eclipse.kura.example.publisher.version=1.2.0
+org.eclipse.kura.example.serial.publisher.version=1.2.0
+org.eclipse.kura.example.ble.tisensortag.version=1.2.0
+org.eclipse.kura.example.ble.tisensortag.dbus.version=1.2.0
+org.eclipse.kura.example.beacon.version=1.2.0
+org.eclipse.kura.example.beacon.scanner.version=1.2.0
+org.eclipse.kura.example.ibeacon.advertiser.version=1.2.0
+org.eclipse.kura.example.ibeacon.scanner.version=1.2.0
+org.eclipse.kura.example.eddystone.advertiser.version=1.2.0
+org.eclipse.kura.example.eddystone.scanner.version=1.2.0
+org.eclipse.kura.example.camel.aggregation.version=1.3.0
+org.eclipse.kura.example.camel.publisher.version=1.2.0
+org.eclipse.kura.example.camel.quickstart.version=1.3.0
+org.eclipse.kura.example.can.version=1.2.0
+org.eclipse.kura.example.gpio.version=1.2.0
+org.eclipse.kura.example.driver.sensehat.version=1.2.0
+org.eclipse.kura.wire.devel.component.provider.version=1.2.0
+org.eclipse.kura.raspberrypi.sensehat.version=1.2.0
+org.eclipse.kura.raspberrypi.sensehat.example.version=1.2.0
+org.eclipse.kura.example.wire.math.singleport.provider.version=1.2.0
+org.eclipse.kura.example.wire.math.multiport.provider.version=1.2.0
+org.eclipse.kura.web2.version=2.2.0
+org.eclipse.kura.wire.camel.version=1.2.0
+org.eclipse.kura.wire.component.provider.version=1.2.0
+org.eclipse.kura.wire.h2db.component.provider.version=2.2.0
+org.eclipse.kura.wire.db.component.provider.version=1.0.0
+org.eclipse.kura.wire.script.filter.provider.version=1.2.0
+org.eclipse.kura.wire.helper.provider.version=1.2.0
+org.eclipse.kura.wire.provider.version=1.2.0
+org.eclipse.kura.rest.provider.version=1.3.0
+org.eclipse.kura.rest.asset.provider.version=1.2.0
+org.eclipse.kura.hook.file.move.provider.version=1.2.0
+org.eclipse.kura.misc.cloudcat.version=1.2.0
+org.eclipse.kura.json.marshaller.unmarshaller.provider.version=1.2.0
+org.eclipse.kura.xml.marshaller.unmarshaller.provider.version=1.2.0
+org.eclipse.kura.wire.component.conditional.provider.version=1.2.0
+org.eclipse.kura.wire.component.join.provider.version=1.2.0
+org.eclipse.kura.cloudconnection.eclipseiot.mqtt.provider.version=1.2.0
+org.eclipse.kura.http.server.manager.version=1.2.0
+org.eclipse.kura.cloudconnection.raw.mqtt.provider.version=1.2.0
+org.eclipse.kura.web2.ext.version=1.2.0
+org.eclipse.kura.useradmin.store.version=1.2.0
+org.eclipse.kura.network.threat.manager.version=1.2.0
+org.eclipse.kura.core.tamper.detection.version=1.2.0
+org.eclipse.kura.log.filesystem.provider.version=1.1.1
```

Comment:
I believe 1.1.1 is wrong here. As far as I know it should be 1.1.0.
The 1.1.1 is probably due to a wrong version update in https://github.com/eclipse/kura/pull/4108